sources/packages/web/src/types/contracts/DataTableContract.ts

```diff
@@ -557,3 +557,144 @@ export const StudentAppealsHistoryHeaders = [
   { title: "Application Number", key: "applicationNumber", sortable: true },
   { title: "Actions", key: "actions", sortable: false },
 ];
+
+/**
+ * Overawards header.
+ */
+export const OverawardsHeaders = [
+  { title: "Date Added", sortable: false, key: "dateAdded" },
+  { title: "Application #", sortable: false, key: "applicationNumber" },
+  { title: "Origin", sortable: false, key: "overawardOrigin" },
+  { title: "Type", sortable: false, key: "assessmentTriggerType" },
+  { title: "Award", sortable: false, key: "awardValueCode" },
+  { title: "Overaward Amount", sortable: false, key: "overawardValue" },
+];
+
+/**
+ * Overawards Deductions header.
+ */
+export const OverawardDeductionsHeaders = [
+  { title: "Date Added", sortable: false, key: "dateAdded" },
+  { title: "Application #", sortable: false, key: "applicationNumber" },
+  { title: "Origin", sortable: false, key: "overawardOrigin" },
+  { title: "Added By", sortable: false, key: "addedByUser" },
+  { title: "Award", sortable: false, key: "awardValueCode" },
+  { title: "Overaward Amount", sortable: false, key: "overawardValue" },
```

Comment:
The last row here must be `Amount deducted`.

Reply:
Fixed.